apps/dde-am/src/main.cpp

```diff
@@ -22,8 +53,19 @@ int main(int argc, char *argv[])
     QCommandLineOption launchedByUserOption("by-user",
                                             "Launched by user, it's useful for counting launched times.");
     parser.addOption(launchedByUserOption);
+    QCommandLineOption desktopPathOption(QStringList() << "d" << "desktop-path",
+                                         "Specify .desktop file path to launch app", "path");
+    parser.addOption(desktopPathOption);
+    QCommandLineOption uriOption(QStringList() << "u" << "uri",
+                                "Specify file URI (e.g.: file:///path/to/app.desktop)", "uri");
+    parser.addOption(uriOption);
+    QCommandLineOption envOption(QStringList() << "e" << "env",
+                                "Set environment variable, format: NAME=VALUE (can be used multiple times)", "env");
+    parser.addOption(envOption);
+    
     parser.addPositionalArgument("appId", "Application's ID.");
     parser.addPositionalArgument("action", "Name of the action identifiers for the application, optionally.");
+    parser.addPositionalArgument("env_var", "Environment variable, format: ENV_VAR=value (recommend using -e option), optionally.");
```

Comment:
```suggestion
    parser.addPositionalArgument("env-var", "Environment variable, format: ENV_VAR=value (recommend using -e option), optionally.");
```

惯用写法是用 `-` 而非 `_`。或者这里直接用 `env` 也行